_data/nil3sh99/nil3sh99.md

```diff
@@ -0,0 +1,5 @@
+name: Nilesh Aggarwal <br/>
+institution: Bharati Vidyapeeth's College of Engineering<br/>
+profile_pic: nilesh.png<br/>
```

Comment:
Incorrect file name, please do one of the following:
1. Change `nilesh.png` to `nilesh.jpg` in the markdown file.
2. Rename `nilesh.jpg` to `nilesh.png`.
I'd suggest you change the markdown file!